### ISSUE TITLE
'dotnet restore3' failing with "Unable to resolve 'Microsoft.NET.Sdk.Web'"

### ISSUE THREAD

**Issue by pealmeid** [dotnet/cli#4490]
## Steps to reproduce

Install SDK version `1.0.0-preview3-003892` and create an "empty" web project:

`mkdir test`
`cd test`
`dotnet new -t web`
`dotnet restore3`

## Expected  behavior
- restoration of all NuGet packages

## Actual behavior
`restore3` fails with the following message: 

`C:\Program Files\dotnet\sdk\1.0.0-preview3-003892\NuGet.targets(70,5): error : Unable to resolve 'Microsoft.NET.Sdk.Web (>= 1.0.0-alpha-20161019-1)' for '.NETCoreApp,Version=v1.0'.`

## Environment data
`dotnet --info` output:

    .NET Command Line Tools (1.0.0-preview3-003892)

    Product Information:
     Version:            1.0.0-preview3-003892
     Commit SHA-1 hash:  5301f08423

    Runtime Environment:
     OS Name:     Windows
     OS Version:  10.0.14393
     OS Platform: Windows
     RID:         win10-x64

**Comment by livarcocc** [comment]
Right now these packages are not published to NuGet.org yet, so you need a NuGet.Config pointing to a feed that contains the packages.

You can look for the one we have in the CLI as an example: https://github.com/dotnet/cli/blob/rel/1.0.0/NuGet.Config

**Comment by pealmeid** [comment]
@livarcocc thanks for the reply, I added the NuGet.Config file as described but I was still getting the error:

`Unable to resolve 'Microsoft.NET.Sdk.Web (>= 1.0.0-alpha-20161019-1)' for '.NETCoreApp,Version=v1.0'.`

So I changed the reference to the package in `test.csproj` to:

    <PackageReference Include="Microsoft.NET.Sdk.Web">
          <Version>1.0.0-alpha-*</Version>
    </PackageReference>

in order to get the latest version of the package and `restore3` was able to resolve it, but now it's failing for another package:

`Unable to resolve 'Microsoft.NET.Sdk.Publish (>= 1.0.0-alpha-20161006)' for '.NETCoreApp,Version=v1.0'.`

I can't find a reference to `Microsoft.NET.Sdk.Publish` in any of the project files, so I'm at a loss regarding how to solve this. Any help is greatly appreciated.

Thanks in advance!
